src/components/add-transfer-dialog.tsx

```
@@ -185,86 +181,91 @@ export function AddTransferDialog({ open, onOpenChange, onSuccess }: AddTransfer
-            <TransferTypeField
-              disabled={isLoading}
+          <TransferTypeField
+            disabled={isLoading}
+            formData={formData}
+            setFormData={setFormData}
+            includeDisposalStyle
+          />
+
+          {formData.loai_hinh === 'thanh_ly' && (
+            <div className="p-3 mt-2 bg-yellow-50 border border-yellow-200 rounded-lg text-sm text-yellow-800">
+              <p><strong>Lưu ý:</strong> Khi chọn "Thanh lý", thiết bị sẽ được chuyển về cho <strong>Tổ QLTB</strong> để xử lý. Sau khi hoàn tất, trạng thái thiết bị sẽ được cập nhật thành "Ngưng sử dụng".</p>
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&`#34`;`, `&rdquo;`.

(react/no-unescaped-entities)